e2e/app/agent/prometheus.go

```diff
@@ -87,6 +93,11 @@ func genPromConfig(ctx context.Context, testnet types.Testnet, secrets Secrets,
 				MetricsPath: "/metrics",
 				targets:     nodeTargets,
 			},
+			{
+				JobName:     "geth",
+				MetricsPath: "/metrics",
```

Comment:
Shouldn't this actually be `/debug/metrics/prometheus`?
Unless I'm missing something, this is what I saw in [the docs](https://geth.ethereum.org/docs/monitoring/metrics#:~:text=We%20also%20provide%20Prometheus%2Dformatted%20metrics%20data).

Reply:
yeah, good catch: `/debug/metrics `